### DISCUSSION TITLE
VSCode `Copilot Chat` extension cannot be installed

### DISCUSSION THREAD

**Discussion by jason-chang** [community/community#55796] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

When I try to install the `Copilot Chat` extension I always get the following popup.

![image](https://github.com/community/community/assets/4367687/2853a2d9-d9ae-4278-8911-0b3265bcd576)

It means: The 'github.copilot-chat' extension cannot be installed because there is no published version.

But when I checked the extension's changelog, I found that there is a published version !!!

![image](https://github.com/community/community/assets/4367687/9599f4d4-b533-4405-a89d-e53a4bd5aeaf)

What happened?

**Comment by Akash1134** [comment, marked as answer]
Hi @jason-chang the chat extension is currently only compatible with vscode-insiders. Please consult the installation instructions in your welcome email.
